Analysis/src/Transpiler.cpp

```diff
@@ -49,7 +49,7 @@ struct Writer
     virtual void keyword(std::string_view) = 0;
     virtual void symbol(std::string_view) = 0;
     virtual void literal(std::string_view) = 0;
-    virtual void string(std::string_view) = 0;
+    virtual void string(std::string_view, bool quoted = true) = 0;
```

Comment:
It would be better to avoid default argument values and update the second call to `string` instead (there are only 2 calls).